microhapulator/data/marker_details_template.html

```diff
@@ -64,36 +69,44 @@ <h1>MicroHapulator Report: Marker Detail</h1>
                 <option>{{markername}}</option>
                 {% endfor %}
             </select>
-            <input id="searchbar" , type="search" onsearch="search_marker()" name="search" placeholder="Search markers..">
-            <p></p>
+            <input id="searchbar" type="search" onsearch="search_marker()" name="search" placeholder="Search markers...">
+            <br />
             {% for markername in markernames %}
             <div class="marker_details" id={{markername}} style="display:none">
-                <p>
                 <h4>Marker Length</h4>
                 <p>{{marker_details_table.loc[markername,'Length']}} bp</p>
-                <br/>
+                <br />
+
                 <h4>%GC content</h4>
                 <p>{{marker_details_table.loc[markername,'GC']}}%</p>
-                <br/>
+                <br />
+
                 <h4>SNP Offsets</h4>
                 <p><strong>Locus</strong>: <span class="offsets">{{marker_details_table.loc[markername,'Offsets']}}</span></p>
                 <p><strong>GRCh38 {{marker_details_table.loc[markername,"Chrom"]}}</strong>: {{marker_details_table.loc[markername,"Hg38Offset"]}}</p>
-                <br/>
+                <br />
+
                 <h4>Marker Sequence (SNPs in red) </h4>
                 <div class="sequence">{{marker_details_table.loc[markername,"Sequence"]}}</div>
-                <br><br><br>
+                <br />
+
+                <h4>Read Mapping</h4>
+                <p>
+                    In this table, "Mapped Reads" refers to the total number of reads that mapped to the marker
+                    reference sequence. "Mapped Rate" refers to the proportion of reads mapped compared to what we would
+                    <em>expect</em> to see with perfectly even coverage across all markers: that is, a "Mapped Rate" of
+                    > 1.0 indicates higher than expected coverage and < 1.0 indicates lower than expected coverage. "Off
+                    Target Reads" and "Off Target Rate" refer to the number and percentage of reads that map to the
+                    marker reference, but preferentially map elsewhere when aligned to the entire genome.
+                </p>
```

Comment:
I also cleaned up the HTML markup a bit: most web browsers are pretty forgiving, but this cleans up a few invalid bits.